app/src/main/java/sensors_in_paradise/sonar/util/PreferencesHelper.kt

```diff
@@ -5,6 +5,7 @@ import android.content.SharedPreferences
 import androidx.camera.video.Quality
 import androidx.preference.PreferenceManager
 import sensors_in_paradise.sonar.R
+import sensors_in_paradise.sonar.page2.camera.pose_estimation.ModelType
```

Comment:
`sensors_in_paradise.sonar.page2` is an invalid package. package `page2` has been moved to `screen_recording` or similar

Reply:
jut aufgepasst!

Reply:
den fehler hab ich extra eingebaut um dich zu testen

Reply:
Sauber